src/jit/Analysis.cpp

```diff
@@ -579,61 +579,61 @@ void JITCompiler::computeOperandLocations(JITFunction* jit_func,
             for (Index i = 0; i < end; ++i) {
                 switch (operand->location.type) {
                 case Operand::Stack: {
-                    stack_allocator->push(operand->location.valueInfo);
-                    operand->value = stack_allocator->values().back().value;
+                    m_stackAllocator->push(operand->location.valueInfo);
+                    operand->value = m_stackAllocator->values().back().value;
                     break;
                 }
                 case Operand::Register: {
-                    stack_allocator->pushReg(operand->value,
+                    m_stackAllocator->pushReg(operand->value,
                                              operand->location.valueInfo);
```

Comment:
Something is wrong with alignment, did you run the style checker?

Reply:
No, I forgot that I did not have it installed on the machine I am working on. I am fixing this right now.